frontend/src/app/north/explore-cache/archive-files/archive-files.component.ts

```diff
@@ -109,4 +115,12 @@ export class ArchiveFilesComponent implements OnInit {
       this.mainArchiveFilesCheckboxState = 'INDETERMINATE';
     }
   }
+
+  changePage(pageNumber: number) {
+    this.archiveFilePages = this.createPage(pageNumber);
+  }
+
+  private createPage(pageNumber: number): Page<any> {
+    return createPageFromArray(this.archiveFiles, PAGE_SIZE, pageNumber);
+  }
```

Comment:
I don't think this method is useful: can you not used createPageFromArray in changePage ?